Apps/W1/File - Azure Blob Service Connector/app/app.json

```diff
@@ -0,0 +1,30 @@
+{
+    "id":  "c9ce86fe-cb70-4b79-be03-d21856b1a4ca",
+    "name":  "File - Azure Blob Service Connector",
+    "publisher":  "Microsoft",
+    "brief":  "Enable all users to use a Azure Blob Storage to save files from Business Central.",
```

Comment:
It's not really what this does. It adds the capability to do file operations on the Azure Blob Storage through the External File Storage module. This brief (and the other two) need a bit more work.